build_docs.pl

```diff
@@ -1113,6 +1119,7 @@ sub check_opts {
         die('--rebuild only compatible with --all') if $Opts->{rebuild};
         die('--reposcache only compatible with --all') if $Opts->{reposcache};
         die('--skiplinkcheck only compatible with --all') if $Opts->{skiplinkcheck};
+        die('--warnlinkcheck only compatible with --all') if $Opts->{warnlinkcheck};
```

Comment:
Down here is probably where you'd want to raise an error of both options are provided.

Reply:
Done!  I added a clause in fdc74aa2d651c62fb9eafa72b7c879489714a1ad